src/Provider.tsx

```diff
@@ -2,8 +2,7 @@ import * as React from "react"
 import { shallowEqual } from "./utils/utils"
 import { IValueMap } from "./inject"
 
-export type TProviderShape = Record<string, any>
-export const MobXProviderContext: React.Context<TProviderShape> = React.createContext({})
+export const MobXProviderContext: React.Context<IValueMap> = React.createContext({})
```

Comment:
Just a little less noisy variant :)
```suggestion
export const MobXProviderContext = React.createContext<IValueMap>({})
```

Reply:
Wow, I feel dumb.